nsxt/data_source_nsxt_policy_edge_node.go

```diff
@@ -34,17 +35,29 @@ func dataSourceNsxtPolicyEdgeNode() *schema.Resource {
 
 func dataSourceNsxtPolicyEdgeNodeRead(d *schema.ResourceData, m interface{}) error {
 	// Read an edge node by name or id
+	edgeClusterPath := d.Get("edge_cluster_path").(string)
+	memberIndex, memberIndexSet := d.GetOkExists("member_index")
```

Comment:
This should be GetOk - I see that GetOkExists doesn't work for non-bools

Reply:
Done